src/esbuild/index.ts

```diff
@@ -112,6 +112,7 @@ export async function runEsbuild(
     format !== 'iife' &&
       externalPlugin({
         external,
+        noExternal: options.external,
```

Comment:
should be `options.noExternal`?

Reply:
```suggestion
        noExternal: options.noExternal,
```

Reply:
Yeah... I did the changes in the gh editor on mobile.